CodeStarterCamp_Week4/main.swift

```diff
@@ -2,11 +2,104 @@
 //  main.swift
 //  CodeStarterCamp_Week4
 //
-//  Created by yagom.
 //  Copyright © yagom academy. All rights reserved.
+//  Created by JeonSangHyeok on 2023/03/15.
 //
 
 import Foundation
 
-print("Hello, World!")
+struct Activity {
+    let name: String
+    let action: (BodyCondition) -> Void
+}
+
+class BodyCondition {
+    var upperBodyStrength: Int = 0 {
+        didSet {
+            print("상체근력이 \(upperBodyStrength - oldValue) 상승합니다.")
+        }
+    }
+    var lowerBodyStrength: Int = 0 {
+        didSet {
+            print("하체근력이 \(lowerBodyStrength - oldValue) 상승합니다.")
+        }
+    }
+    var muscularEndurance: Int = 0 {
+        didSet {
+            print("근지구력이 \(muscularEndurance - oldValue) 상승합니다.")
+        }
+    }
+    var fatigue: Int = 0 {
+        didSet {
+            if fatigue >= oldValue {
+                print("피로도가 \(fatigue - oldValue) 상승합니다.")
+            } else {
+                print("피로도가 \(fatigue - oldValue) 감소합니다.")
+            }
```

Comment:
```suggestion
            fatigue >= oldValue ? print("피로도가 \(fatigue - oldValue) 상승합니다.") : print("피로도가 \(fatigue - oldValue) 감소합니다.")
```
`didSet문`에 `if문`까지 들어오면 코드의 뎁스가 길어져서 저는 보통 `삼항연산자`를 활용해요!

Reply:
네, 삼항연산자로 활용하여 수정할게요!